cicd/pipelines/viewing-pipeline-logs-using-the-openshift-logging-operator.adoc

```diff
@@ -0,0 +1,33 @@
+[id='viewing-pipeline-logs-using-the-openshift-logging-operator']
```

Comment:
```suggestion
[id="viewing-pipeline-logs-using-the-openshift-logging-operator"]
```
See guidelines: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assemblymodule-titles-and-section-headings